lib/rails/diff.rb

```diff
@@ -61,6 +61,18 @@ def rails_path
         end
       end
 
+      def railsrc_path
+        "#{ENV["HOME"]}/.railsrc"
+      end
+
+      def railsrc_options
+        @railsrc_options ||= begin
+          return unless File.exist?(railsrc_path)
+
+          File.read(railsrc_path).tr("\n", " ")
+        end
```

Comment:
Sorry for being pedant here. I think this will call `File.exist?` twice because ruby doesn't memoize nil values:

```suggestion
        return @railsrc_options if defined?(@railsrc_options)
  
        @railsrc_options = File.read(railsrc_path).tr("\n", " ") if File.exist?(railsrc_path)
```

Reply:
Good catch!